master/static/migrations/20211102110152_make_model_registry_names_unique.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE public.models ADD CONSTRAINT uni_name UNIQUE (name);
```

Comment:
This seems pretty unsafe to me now.

I'm concerned that we shipped a release where we let users violate this and now the cat is out-of-the-bag, so-to-speak.

Reply:
it's safe because of the dml above, but i'm not sure that is OK without a much bigger warning that we may be renaming things without warning.

also, constraint names are global I think, so name this something like `models_name_unique` (the norm is `{table_name}_{column_name}_unique`.

Reply:
i'm inclined to add a big warning and do the DML, though, so names as non-unique is so counter intuitive and i doubt it affects many users.